proto/storage/compliance_operator_v2.proto

```diff
@@ -105,7 +109,7 @@ message ComplianceOperatorScanConfigurationV2 {
 message ComplianceOperatorClusterScanConfigStatus {
     string                        id                  = 6 [(gogoproto.moretags) = 'sql:"pk,type(uuid)"'];
     string                        cluster_id          = 1 [(gogoproto.moretags) = 'search:"Cluster ID,hidden" sql:"fk(Cluster:id),no-fk-constraint,type(uuid)"'];
-    string                        scan_id             = 2 [(gogoproto.moretags) = 'search:"Compliance Scan Config ID,hidden" sql:"fk(ComplianceOperatorScanConfigurationV2:id),no-fk-constraint"'];
+    string                        scan_config_id      = 2 [(gogoproto.moretags) = 'search:"Compliance Scan Config ID,hidden" sql:"fk(ComplianceOperatorScanConfigurationV2:id),no-fk-constraint,type(uuid)"'];
```

Comment:
technically it's a breaking change. Can we do it or should we mark `scan_id 2`  as reserved?

Reply:
@janisz It is but it isn't.  These tables have not been deployed anywhere yet.  Which is the only reason I bothered with making this change.